projects/sunbird-video-player/src/lib/sunbird-video-player.component.ts

```diff
@@ -35,6 +36,7 @@ export class SunbirdVideoPlayerComponent implements OnInit, AfterViewInit, OnDes
   showQumlPlayer = false;
   QumlPlayerConfig: any = {};
   videoInstance: any;
+  transcripts: Transcripts;
```

Comment:
It's already there as part of this.playerConfig. why do you need one more vaiable here.
Remove it if you used only once or twice.

Reply:
Done